<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SharedNote</class>
 <widget class="QMainWindow" name="SharedNote">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shared Note Editor</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/resources/images/sharedEditor.png</normaloff>:/resources/images/sharedEditor.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <widget class="QWidget" name="m_documentSupport" native="true">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>758</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuEdit_Syntax_Highlighting">
     <property name="title">
      <string>Syntax Highlighting...</string>
     </property>
     <addaction name="m_highlightMarkdownAction"/>
    </widget>
    <addaction name="actionEdit_Undo"/>
    <addaction name="actionEdit_Redo"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Cut"/>
    <addaction name="actionEdit_Copy"/>
    <addaction name="actionEdit_Paste"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Find_All"/>
    <addaction name="actionEdit_Find"/>
    <addaction name="menuEdit_Syntax_Highlighting"/>
    <addaction name="separator"/>
    <addaction name="m_markdownPreview"/>
    <addaction name="actionFile_Print"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionView_Line_Wrap"/>
    <addaction name="m_showParticipants"/>
   </widget>
   <widget class="QMenu" name="menuText">
    <property name="title">
     <string>Text</string>
    </property>
    <addaction name="actionText_Shift_Left"/>
    <addaction name="actionText_Shift_Right"/>
    <addaction name="actionText_Comment_Line"/>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuText"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFile_Print"/>
   <addaction name="actionEdit_Undo"/>
   <addaction name="actionEdit_Redo"/>
   <addaction name="actionEdit_Cut"/>
   <addaction name="actionEdit_Copy"/>
   <addaction name="actionEdit_Paste"/>
   <addaction name="actionHelp_How_to_Collaborate"/>
   <addaction name="separator"/>
   <addaction name="m_refresh"/>
   <addaction name="separator"/>
   <addaction name="m_markdownPreview"/>
  </widget>
  <action name="actionEdit_Undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="undo">
     <normaloff>:/sharednote/resources/images/undo.png</normaloff>:/sharednote/resources/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="redo">
     <normaloff>:/sharednote/resources/images/redo.png</normaloff>:/sharednote/resources/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Cut">
   <property name="icon">
    <iconset theme="cut">
     <normaloff>:/sharednote/resources/images/cut.png</normaloff>:/sharednote/resources/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Copy">
   <property name="icon">
    <iconset theme="copy">
     <normaloff>:/sharednote/resources/images/copy.png</normaloff>:/sharednote/resources/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Paste">
   <property name="icon">
    <iconset theme="paste">
     <normaloff>:/sharednote/resources/images/paste.png</normaloff>:/sharednote/resources/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEdit_Find">
   <property name="text">
    <string>Find...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionView_Line_Wrap">
   <property name="text">
    <string>Line Wrap</string>
   </property>
  </action>
  <action name="m_highlightMarkdownAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Markdown</string>
   </property>
  </action>
  <action name="actionText_Shift_Left">
   <property name="text">
    <string>Shift Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+[</string>
   </property>
  </action>
  <action name="actionText_Shift_Right">
   <property name="text">
    <string>Shift Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+]</string>
   </property>
  </action>
  <action name="actionText_Comment_Line">
   <property name="text">
    <string>Un/Comment Line</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+/</string>
   </property>
  </action>
  <action name="actionFile_Save_A_Copy_As">
   <property name="text">
    <string>Save A Copy As...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFile_Save_All">
   <property name="text">
    <string>Save All</string>
   </property>
  </action>
  <action name="actionTools_Announce_Document">
   <property name="text">
    <string>Announce Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="m_showParticipants">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show/Hide Participants</string>
   </property>
  </action>
  <action name="actionView_Hide_Show_Chat">
   <property name="text">
    <string>Show/Hide Chat</string>
   </property>
  </action>
  <action name="actionTools_Connect_to_Document">
   <property name="text">
    <string>Connect to Document...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionWindow_Split">
   <property name="text">
    <string>Split</string>
   </property>
  </action>
  <action name="actionTools_Resynchronize_Document">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Resynchronize Document</string>
   </property>
  </action>
  <action name="actionWindow_Next_Document">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Next Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+]</string>
   </property>
  </action>
  <action name="actionWindow_Previous_Document">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Previous Document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+[</string>
   </property>
  </action>
  <action name="actionEdit_None">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>None</string>
   </property>
  </action>
  <action name="actionWindow_Remove_Split">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Remove Split</string>
   </property>
  </action>
  <action name="actionWindow_Split_Side_by_Side">
   <property name="text">
    <string>Split Side by Side</string>
   </property>
  </action>
  <action name="actionTools_Preferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionEdit_Find_All">
   <property name="text">
    <string>Find All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionHelp_About_Cahoots">
   <property name="text">
    <string>About Cahoots</string>
   </property>
  </action>
  <action name="actionHelp_About_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionHelp_How_to_Collaborate">
   <property name="icon">
    <iconset theme="help">
     <normaloff>:/sharednote/resources/images/help.png</normaloff>:/sharednote/resources/images/help.png</iconset>
   </property>
   <property name="text">
    <string>How to Collaborate</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionFile_Print">
   <property name="icon">
    <iconset theme="print">
     <normaloff>:/sharednote/resources/images/print.png</normaloff>:/sharednote/resources/images/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="m_refresh">
   <property name="icon">
    <iconset theme="action-rotate">
     <normaloff>:/resources/images/action-rotate.png</normaloff>:/resources/images/action-rotate.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="m_markdownPreview">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="preview_markdown">
     <normaloff>:/resources/images/preview_markdown.svg</normaloff>:/resources/images/preview_markdown.svg</iconset>
   </property>
   <property name="text">
    <string>Markdown Preview</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
